docs/operation_semantics.md

```diff
@@ -1400,31 +1380,65 @@ element of `operand`.
 
 <b>`RoundNearestEven(operand)`</b> Element-wise rounding, ties to nearest even.
 
-<b>`Rsqrt(operand)`</b> Element-wise reciprocal of square root operation
-`x -> 1.0 / sqrt(x)`.
-
 <b>`Sign(operand)`</b> Element-wise sign operation `x -> sgn(x)` where
 
 $$\text{sgn}(x) = \begin{cases} -1 & x < 0\\ -0 & x = -0\\ NaN & x = NaN\\ +0 &
 x = +0\\ 1 & x > 0 \end{cases}$$
 
 using the comparison operator of the element type of `operand`.
 
-<b>`Sin(operand)`</b> Element-wise sine `x -> sin(x)`.
+| Arguments | Type    | Semantics                   |
+| --------- | ------- | --------------------------- |
+| `operand` | `XlaOp` | The operand to the function |
 
-<b>`Sqrt(operand)`</b> Element-wise square root operation `x -> sqrt(x)`.
+The function is applied to each element in the `operand` array, resulting in an
+array with the same shape. It is allowed for `operand` to be a scalar
+(0-dimensional).
 
-<b>`Tan(operand)`</b> Element-wise tangent `x -> tan(x)`.
+### Optional Result Accuracy
 
-<b>`Tanh(operand)`</b> Element-wise hyperbolic tangent `x -> tanh(x)`.
+XlaBuilder supports these element-wise unary functions with the optional `result_accuracy` argument:
 
-Arguments | Type    | Semantics
---------- | ------- | ---------------------------
-`operand` | `XlaOp` | The operand to the function
+<b>`Cbrt(operand, result_accuracy)`</b> Element-wise cubic root operation `x -> cbrt(x)`.
 
-The function is applied to each element in the `operand` array, resulting in an
-array with the same shape. It is allowed for `operand` to be a scalar
-(0-dimensional).
+<b>`Cos(operand, result_accuracy)`</b> Element-wise cosine `x -> cos(x)`.
```

Comment:
Lets maybe duplicate these unary ops - keep them above with the list of unary ops since very few people are using result accuracy / most only care about them in the unary ops section. But in this new section "Optional Result Accuracy" lets duplicate the subset of unary operations that support result accuracy.